### PR TITLE
Updates to the callbacks

### PR DESCRIPTION
**Pull request by peter-nexmo**
## Description

The callbacks sent as stages of a Voice API have been enhanced, and have been updated to reflect the current format.

### CODE REVIEW

File: _documentation/voice/voice-api/guides/handle-call-status.md [hunk 1]
```diff
@@ -17,10 +17,13 @@ A call can be initiated via a request to [`POST /calls`](/api/voice#payload). Th
 
 ```json
 {
+  "from": "447700900000",
+  "to":  "447700900001",
   "uuid": "ade7b1d3-6cb7-4350-8750-dfdee725f256",
+  "conversation_uuid": "CON-17f767a5-c1d7-4e9e-a132-6b3f3289efff"
```
Comment (adambutler) [inline _documentation/voice/voice-api/guides/handle-call-status.md]: - [ ] Missing a comma at the end of this line.
- [ ] Change UUIDs to `aaaaaaaa-bbbb-cccc-dddd-0123456789ab` & `CON-bbbbbbbb-cccc-dddd-eeee-0123456789ab`

File: _documentation/voice/voice-api/guides/handle-call-status.md [hunk 2]
```diff
@@ -17,10 +17,13 @@ A call can be initiated via a request to [`POST /calls`](/api/voice#payload). Th
...
   "status": "started",
   "direction": "outbound",
-  "conversation_uuid": "17f767a5-c1d7-4e9e-a132-6b3f3289efff"
+  "timestamp": "2018-01-01T12:34:56.789Z"
```
Comment (adambutler) [inline _documentation/voice/voice-api/guides/handle-call-status.md]: - [ ] Change timestamp to `2020-01-01T14:00:00.000Z`

File: _examples/voice/guides/handle-call-status/ncco/call-answered.md [hunk 3]
```diff
@@ -10,9 +10,10 @@ menu_weight: 2
   "from": "447700900000",
   "to": "447700900001",
   "uuid": "7dd82abd-563e-4b0a-ba17-e5f0297a7c63",
-  "conversation_uuid": "78fd94b9-19f0-4df1-a538-8b0eebb09a18",
+  "conversation_uuid": "CON-78fd94b9-19f0-4df1-a538-8b0eebb09a18",
```
Comment (adambutler) [inline _examples/voice/guides/handle-call-status/ncco/call-answered.md]: - [ ] Change UUID to `CON-bbbbbbbb-cccc-dddd-eeee-0123456789a`

File: _examples/voice/guides/handle-call-status/ncco/call-answered.md [hunk 4]
```diff
@@ -10,9 +10,10 @@ menu_weight: 2
...
   "status": "answered",
   "direction": "outbound",
-  "network": null
+  "network": null,
+  "timestamp":"2018-01-01T12:34:56.789Z"
```
Comment (adambutler) [inline _examples/voice/guides/handle-call-status/ncco/call-answered.md]: - [ ] Change timestamp to `2020-01-01T14:00:00.000Z`

File: _examples/voice/guides/handle-call-status/ncco/call-complete.md [hunk 5]
```diff
@@ -13,9 +13,10 @@ menu_weight: 3
   "from": "447700900000",
   "to": "447700900001",
   "uuid": "19d4df7c-d0dd-40c5-8460-1e88bd5d0d6b",
-  "conversation_uuid": "2472b52f-fcaa-4306-b4cb-621c14534020",
+  "conversation_uuid": "CON-2472b52f-fcaa-4306-b4cb-621c14534020",
```
Comment (adambutler) [inline _examples/voice/guides/handle-call-status/ncco/call-complete.md]: - [ ] Change UUID to `CON-bbbbbbbb-cccc-dddd-eeee-0123456789a`

File: _examples/voice/guides/handle-call-status/ncco/call-complete.md [hunk 6]
```diff
@@ -13,9 +13,10 @@ menu_weight: 3
...
   "status": "completed",
-  "direction": "out",
-  "network": "23433"
+  "direction": "outbound",
+  "network": "23433",
```
Comment (adambutler) [inline _examples/voice/guides/handle-call-status/ncco/call-complete.md]: - [ ] Change Network to `12345`

File: _examples/voice/guides/handle-call-status/ncco/call-ringing.md [hunk 7]
```diff
@@ -5,9 +5,12 @@ menu_weight: 1
 
 ```json
 {
+  "from": "447700900000",
+  "to": "447700900001",
   "uuid": "7dd82abd-563e-4b0a-ba17-e5f0297a7c63",
-  "conversation_uuid": "78fd94b9-19f0-4df1-a538-8b0eebb09a18",
+  "conversation_uuid": "CON-78fd94b9-19f0-4df1-a538-8b0eebb09a18",
```
Comment (adambutler) [inline _examples/voice/guides/handle-call-status/ncco/call-ringing.md]: - [ ] Change UUID to `CON-bbbbbbbb-cccc-dddd-eeee-0123456789a`

File: _examples/voice/guides/handle-call-status/ncco/call-ringing.md [hunk 8]
```diff
@@ -5,9 +5,12 @@ menu_weight: 1
...
   "status": "ringing",
-  "direction": "outbound"
+  "direction": "outbound",
+  "timestamp": "2018-01-01T12:34:56.789Z"
```
Comment (adambutler) [inline _examples/voice/guides/handle-call-status/ncco/call-ringing.md]: - [ ] Change timestamp to `2020-01-01T14:00:00.000Z`

File: _documentation/voice/voice-api/guides/handle-call-status.md [hunk 9]
```diff
@@ -52,3 +55,4 @@ Key | Value
 `price` | The total price charged for this Call.
 `duration` | The time elapsed for the Call to take place in seconds.
 `end_time` | The time the Call ended in the following format: YYYY-MM-DD HH:MM:SS. For example, 2020-01-01 12:00:00.
+`timestamp` | The time the callback was created in the following format: YYYY-MM-DD HH:MM:SS. For example, 2020-01-01 12:00:00.
```
Comment (leggetter) [inline _documentation/voice/voice-api/guides/handle-call-status.md]: Should this example have millis and z in it?

Comment (peter-nexmo) [reply _documentation/voice/voice-api/guides/handle-call-status.md]: I would prefer it to. I have copied the existing description format from `end_time`.

Comment (adambutler) [reply _documentation/voice/voice-api/guides/handle-call-status.md]: Checked both have the format `2020-01-01T14:00:00.000Z`. Changed description to link to ISO 8601.

**Pull request merged**